cmd/server/main.go

```diff
@@ -14,16 +16,39 @@ const (
 )
 
 func main() {
-	lis, err := net.Listen("tcp", port)
-	if err != nil {
-		log.Fatalf("failed to listen: %v", err)
+	app := cli.NewApp()
+	app.Name = "ocic"
+	app.Usage = "client for ocid"
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "sandboxdir",
+			Value: "/var/lib/ocid/sandbox",
+			Usage: "ocid pod sandbox dir",
+		},
 	}
-	s := grpc.NewServer()
-	service, err := server.New("")
-	if err != nil {
+
+	app.Action = func(c *cli.Context) error {
+		lis, err := net.Listen("tcp", port)
+		if err != nil {
+			log.Fatalf("failed to listen: %v", err)
+		}
+
+		s := grpc.NewServer()
+
+		sandboxDir := c.String("sandboxdir")
+		service, err := server.New("", sandboxDir)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		runtime.RegisterRuntimeServiceServer(s, service)
+		runtime.RegisterImageServiceServer(s, service)
```

Comment:
Thanks for pointing out.
I have not reached that far yet. Currently, I am just trying to process each field inside `PodSandboxConfig`.